src/SourceTerms/Deleptonization/CPU_Src_Deleptonization.cpp

```diff
@@ -159,6 +159,8 @@ static void Src_Deleptonization( real fluid[], const real B[],
    const real Delep_minDens_CGS = (real)1.0e6; // [g/cm^3]
 
 
+   const real MeV2Kelvin        = 1.0/Kelvin2MeV;
```

Comment:
I think this is unnecessary since `MeV2Kelvin` is used one time only. If you still want this, correct the style.